processing/src/main/java/io/druid/guice/ExtensionsConfig.java

```diff
@@ -33,10 +34,6 @@
   @NotNull
   private boolean searchCurrentClassloader = true;
 
-  @JsonProperty
-  @NotNull
-  private List<String> coordinates = ImmutableList.of();
```

Comment:
Can this actually be left in, and give an error at startup if it is specified?

There doesn't seem to be a migration plan overall here. (or I could have just missed it)